### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by kauana**
### Summary of changes
We've made some updates to some of our README files which include:
* Add a fancy new diagram with routecontroller architecture 🤩
* Include information on RouteController
* Include some basic information about our usage of Istio
* Remove references to cfroutesync and bbl
* Add information on scaling/upgrade tests to the CONTRIBUTING guide

For more info, check the tracker story: [#173044078](https://www.pivotaltracker.com/story/show/173044078)

### Additional Context
We are looking for some feedback on our READMEs. Is there any other information we should add? Is everything accurate?

### Acceptance Steps
1. **When** I open a README file
2. **Then** I see up to date information! :tada: 

_Tag your pair, your PM, and/or team_
@cloudfoundry/cf-networking

### CODE REVIEW

**Comment by rosenhouse** [comment]
tiny nit pick on the new diagram: you have an arrow from "K8s API" to "RouteController" with the verb "read". 

I think I'd find it clearer if I saw an arrow from RouteController to K8s API that said "watch"

(I know the diagram it is replacing has something similar from k8s api to metacontroller, and I regret my involvement in that :-) )

File: test/acceptance/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To run tests you need to have the following installed:
   ```bash
   $ wget -O- https://k14s.io/install.sh | bash
   ```
-  
+
```
Comment (XanderStrike) [inline test/acceptance/README.md:12]: 👾 👾 👾 👾 

Probably fine tho

**Review by XanderStrike — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,9 +55,31 @@ using Docker.
 
 
 ### Running Tests
```
Comment (christianang) [inline CONTRIBUTING.md:57]: Maybe worth adding something about what tests we would expect someone to run. E.g I wouldn't expect anyone contributing to run the uptime/scale tests themselves, but we would expect them to run the unit/integration tests. Maybe the acceptance tests?

File: test/acceptance/README.md (hunk 1 above)
Comment (kauana) [reply test/acceptance/README.md:12]: good catch!!! :grinning:

Comment (tcdowney) [reply test/acceptance/README.md:12]: These spaces were there originally for what it's worth. We just removed the trailing whitespace. 😅 

https://raw.githubusercontent.com/cloudfoundry/cf-k8s-networking/179bf56502986aab27a98df8f7f33b0627e564e5/test/acceptance/README.md

**Review by rosenhouse — APPROVED** [review]

**Pull request merged**